Formula/r/rpds-py.rb

```diff
@@ -0,0 +1,73 @@
+class RpdsPy < Formula
+  include Language::Python::Virtualenv
+
+  desc "Python bindings to Rust's persistent data structures"
+  homepage "https://rpds.readthedocs.io/en/latest/"
+  url "https://files.pythonhosted.org/packages/01/80/cce854d0921ff2f0a9fa831ba3ad3c65cee3a46711addf39a2af52df2cfd/rpds_py-0.22.3.tar.gz"
+  sha256 "e32fee8ab45d3c2db6da19a5323bc3362237c8b653c70194414b892fd06a080d"
+  license "MIT"
+
+  depends_on "maturin" => :build
+  depends_on "python@3.11" => [:build, :test]
+  depends_on "python@3.12" => [:build, :test]
```

Comment:
Why not the latest?

Reply:
I was under the impression that this was the expected combination of python versions, from other formulae.

Reply:
@SMillerDev  added the most recent python version